### ISSUE TITLE
Typo in label

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4199]
https://hpo.jax.org/app/browse/term/HP:0010979
